src/javascript/app/pages/trade/digit_ticker.js

```diff
@@ -208,7 +208,7 @@ const DigitTicker = (() => {
         el_container.classList.remove('invisible');
         adjustBoxSizes();
         current_spot = quote.substr(-1);
-        if (el_mask) el_mask.innerText = `${current_tick_count} / ${total_tick_count}`;
+        if (el_mask) el_mask.innerText = `${current_tick_count}/${total_tick_count}`;
```

Comment:
could you undo these spacing changes please? let's keep it same as master if possible

Reply:
I've removed it as per the design, but we could revert. 👍

Reply:
oh, right, it's in string :) let's keep them, my bad